portfolio/src/main/webapp-local/src/components/Projects/Projects.css

```diff
@@ -5,3 +5,7 @@
     margin-bottom: 3rem;
     margin-top: 2rem;
 }
+
+.item {
+margin-top: 45px !important; 
+}
```

Comment:
Because of <legacy reasons I can't remember but we can find out if you're interested>, a lot of tools will complain like this if you don't have a newline at the end of your files. It doesn't actually hurt anything, but you can add one here if you want to get rid of the warning github adds.

Reply:
I had no idea! Thank you for the knowledge 👍